frappe/core/doctype/system_settings/system_settings.py

```diff
@@ -82,7 +83,6 @@ class SystemSettings(Document):
 		]
 		otp_issuer_name: DF.Data | None
 		password_reset_limit: DF.Int
-		rate_limit_email_link_login: DF.Int
```

Comment:
This got wrongly dropped in a previous PR, not sure why.
Fixed in #27366, can you update/rebase this branch now